ci/gpu/build.sh

```diff
@@ -73,7 +74,7 @@ conda list
 ################################################################################
 
 logger "Build libcuml..."
-$WORKSPACE/build.sh clean libcuml cuml prims -v
+$WORKSPACE/build.sh clean libcuml cuml prims --multigpu -v
```

Comment:
Do we need something that tests that we don't break building _without_ --multigpu? (can wait till after release)

Reply:
#1001